eng/Generate.ps1

```diff
@@ -57,7 +58,7 @@ function Add-CadlRanch-Cadl([string]$testName, [string]$projectPrefix, [string]$
     if ($cadlFolders) {
         $cadlFolder = $cadlFolders[0]
         $cadlMain = Join-Path $cadlFolder "main.cadl"
-        Add-Cadl "$projectPrefix$testName" $projectDirectory $cadlMain
+        Add-Cadl "$projectPrefix$testName" $projectDirectory $cadlMain "--option @azure-tools/cadl-csharp.generateConvenienceAPI=true"
```

Comment:
I think this is controlled inside cadl file. why do we need this flag?

Reply:
It's a workaround for caldRanch projects. Current Convenience decorator has not be applied into cadlRanch project, so we set the flag as now, when we add Convenience decorators, we will remove this workaround.

Reply:
Better to add a 'TODO' here as this is a temp solution.